src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -38,6 +38,12 @@ export type OnMoveTextNodeCallbackType = (
 ) => void;
 
 export type OnSelectNodeCallbackType = (equipmentId: string, nodeId: string) => void;
+export type OnToggleNADHoverCallbackType = (
```

Comment:
Acronym should not have full upper case in variable/class 
```suggestion
export type OnToggleNadHoverCallbackType = (
```